projects/igniteui-angular/src/lib/grids/grid-base.component.ts

```diff
@@ -2350,23 +2353,6 @@ export abstract class IgxGridBaseComponent extends DisplayDensityBase implements
             this.cdr.detectChanges();
             this.parentVirtDir.onChunkLoad.emit(this.headerContainer.state);
         });
-
-        if (this.allowFiltering && this.filterMode === FilterMode.excelStyleFilter) {
-            this.closeExcelStyleDialog();
-        }
-    }
-
-    private closeExcelStyleDialog() {
-        const excelStyleMenu = this.outlet.nativeElement.getElementsByClassName('igx-excel-filter__menu igx-toggle')[0];
```

Comment:
Would love for this to go away (😄), BUT in the Grid now the excel filter dialog can be opened and the grid scrolled horizontally and it just stays there.
Don't think that's intended.

Reply:
OK.